queries.sql

```diff
@@ -153,3 +153,22 @@ SELECT species.name, count(*)
   LEFT JOIN vets ON vets.id = visits.vets_id
   WHERE vets.name = 'Maisy Smith'
   GROUP BY species.name;
+
+
+ /*Pair programing*/
+ /*Before queries - te times of execusion are more than 1000 ms*/
+ 
+ EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+ EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+ EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
+ 
+ /*Apply Indexes to improbe the preformance*/
+ 
+ CREATE INDEX animals_id_index ON visits(animal_id ASC);
+ CREATE INDEX vets_id_index ON visits(vet_id, animal_id, date_of_visit);
+ CREATE INDEX email_index ON owners(email ASC);
```

Comment:
- [ ] Well done implementing the solution to low the execution time, however, one of the basic solutions for the performance problems is adding indexes for FOREIGN KEYS or on columns frequently used in queries. Please add the `CREATE INDEX` lines on the schema.sql file. 👇🏽 

> Everything that creates or change the tables structure should be saved in the schema.sql file.
> Everything that populates tables with data should be saved in the data.sql file.
> Everything that is a query that helps to analyze data should be saved in the queries.sql file.

Reply:
Ok, I will change, thank you